### PR TITLE
Improve the email alerts for the hypertension fast track pilot

### PR DESCRIPTION
**Pull request by NatashaJIbrahim**
## Description of change
This pr does two things to improve the emails being sent in relation to the Hypertension Fast Track Pilot 
       - Modifies the email in the worker to be specifically for engineers to help with
          debugging.
        - Adds a proper email for the stakeholders for tracking the
          submissions that go through the fast track pilot.

### CODE REVIEW

File: app/workers/fast_track/disability_compensation_job.rb [hunk 1]
```diff
@@ -42,14 +42,16 @@ def perform(form526_submission_id, full_name)
 
     private
 
-    def send_fast_track_stakeholder_email_for_testing(form526_submission_id)
+    def send_fast_track_engineer_email_for_testing(form526_submission_id)
       # TODO: This should be removed once we have basic metrics
       # on this feature and the visibility is imporved.
+      body = "A claim was just submitted on the #{Rails.env} environment " \
+             "with submission id: #{form526_submission_id} and job_id #{jid}"
```
Comment (thilton-oddball) [inline app/workers/fast_track/disability_compensation_job.rb:49]: I'm not seeing where jid is coming from.

Comment (NatashaJIbrahim) [reply app/workers/fast_track/disability_compensation_job.rb]: good catch! I just fixed it. It should have been `self.jid`

Comment (thilton-oddball) [reply app/workers/fast_track/disability_compensation_job.rb]: Hmm, it looks like self. isn't actually needed. My bad.

**Review by thilton-oddball — DISMISSED** [review]

**Comment by NatashaJIbrahim** [comment]
This was merged in another branch since this branch was being annoying.

**Pull request closed without merging**